docker/action.yml

```diff
@@ -0,0 +1,8 @@
+name: 'Release Drafter'
+description: 'Drafts your next release notes as pull requests are merged into master.'
+runs:
+  using: 'docker'
+  image: '../Dockerfile'
```

Comment:
😱 this just works out of the box, I was surprised that the workdir defaults to repo!

Awesome job @github 😍
🎉 @TimonVS

Reply:
https://github.com/Jetersen/test-release-drafter-action-adapter/commit/a3a3381132d1c87d380dfd1ae2d46eb6fb107314